app/models/service_template.rb

```diff
@@ -258,6 +258,11 @@ def self.default_reconfiguration_entry_point
     nil
   end
 
+  def set_resource_actions(config_info, dialog)
```

Comment:
I am not very convinced that we want this method in `ServiceTemplate`, especially here the keys in `config_info` are hard codes, obviously inherited from API controller. If we really want it here, can you document the use of this method?
@gmcculloug any thoughts?